specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2021-03-01-preview/nova.json

```diff
@@ -0,0 +1,1143 @@
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/clusterNameParameter"
+          },
+          {
+            "$ref": "#/parameters/dataCenterNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted. Deletion will complete asynchronously."
+          },
+          "204": {
+            "description": "No such cluster to delete."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "cosmos-db.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "put": {
+        "operationId": "CassandraDataCenter_CreateUpdate",
+        "description": "Create or update a managed Cassandra data center. When updating, overwrite all properties. To update only some properties, use PATCH.",
+        "x-ms-examples": {
+          "CosmosDBManagedCassandraDataCenterCreate": {
+            "$ref": "./examples/CosmosDBManagedCassandraDataCenterCreate.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/clusterNameParameter"
+          },
+          {
+            "$ref": "#/parameters/dataCenterNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/DataCenterResource"
+            },
+            "description": "Parameters specifying the managed Cassandra data center."
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Accepted. The data center will be created asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/DataCenterResource"
+            }
+          },
+          "200": {
+            "description": "Accepted. The data center will be updated asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/DataCenterResource"
+            }
+          },
```

Comment:
Are you sure this is correct ?

Reply:
Yes. In https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#put-resource it says:

"The response body should contain at least the original request that was PUT (and any other properties that would be returned in a GET, such as provisioningState, name, Id and type)."